test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/Preseed default - Ansible.snap.txt

```diff
@@ -55,7 +55,7 @@ user-setup-udeb passwd/make-user boolean false
 tasksel tasksel/first multiselect minimal, ssh-server, openssh-server
 
 # Install some base packages
-d-i pkgsel/include string lsb-release wget python
+d-i pkgsel/include string lsb-release wget
```

Comment:
Now i think this change might be unexpected

Reply:
Hmm how about a new issue/commit into this PR?

Reply:
Please open a separate pr for it and keep the changes in this one to what is needed for fixing the foreman_url params.

Reply:
this change will fail right now and isn't related to this PR, please revert it